shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -86,6 +84,19 @@ case class Arrival(Operator: Option[Operator],
     }
   }
 
+  def displayStatusMobile: ArrivalStatus = {
+    (this.Estimated, this.ActualChox, this.Actual) match {
+      case (_, _, _) if isCancelledStatus(this.Status.description.toLowerCase) => ArrivalStatus("Can")
```

Comment:
These abbreviations might be easier to understand

Cancelled -> CNX (https://www.allacronyms.com/CNX/Cancelled)
Diverted -> Dvtd
Delayed -> DLA (https://www.allacronyms.com/DLA/Delayed)